components/StdHeader.vue

```diff
@@ -40,27 +40,60 @@
             <div class="divider"></div>
 
             <div class="mx-10 mt-7 text-blue-800">
-              <a href="">Log Out</a>
+              <a href="#" @click="logout">Log Out</a>
              </div>
           </div>
 
-          <div class="flex ml-auto mx-10 mt-7 text-blue-800" v-if="!this.auth">
-              <a href="">Login</a>
+          <div class="flex ml-auto mx-10 mt-7 text-blue-800" v-if="!authModule.isAuthenticated">
+              <a href="#" @click="login">Login</a> 
              </div>
 
       </div>
   </nav>
 </template>
 
-<script>
-export default{
-  data(){
-    return{
-      auth: true
+<script setup>
+import SvgIcon from '@jamescoyle/vue-icon';
```

Comment:
1. The npm page for this package suggests using https://github.com/Pictogrammers/svg-icon instead.

2. In order to keep this dependency local, consider forking the repo to bcgov so that we still have access to it if the author decides to withdraw the original version. Similar to this:
https://github.com/bcgov/vue-test-utils-helpers

Reply:
done